tools/cmake/cmbuild

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+# make like wrapper around "cmake --build"
+# (c) 2020 Axel Kohlmeyer <akohlmey@temple.edu>
+# This file is in the public domain
+
+WORKDIR="${PWD}"
+PARAMS=""
+MYARGS=""
+
+usage()
+{
+   echo "Usage: cmbuild [-v] [-h] [-C <dir>] [-j <num>] [<target1> [<target2>] ...]" >&2
```

Comment:
I think multiple target are only supported in >=cmake-3.15

Reply:
ok. taking that out and advertise only a single target. will have to test across platforms like centos and ubuntu LTS anyway.